packages/components/app/styles/components/dropdown.scss

```diff
@@ -532,13 +532,16 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
 
     &[disabled] ~ .hds-dropdown-list-item__label,
```

Comment:
now that the `hds-dropdown-list-item__label` is a parent of the `hds-dropdown-list-item__control` this line doesn't make sense anymore
```suggestion
```